.circleci/config.yml

```diff
@@ -43,7 +43,7 @@ jobs:
     working_directory: /home/circleci/go/src/github.com/influxdata/influx-cli
     steps:
       - checkout
-      - run: make influx
+      - run: make crossbuild
```

Comment:
`crossbuild` was _very_ fast for me locally, checking if it's the same in CI (if not, we might want to split into parallelized jobs)

Reply:
`16s` for this vs. `2s` for the last `make influx` to run on the `main` branch